src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -1,128 +1,304 @@
+    <li>From the **VPC** dropdown, select the VPC you chose in Step 4.</li>
+    <li><ul><li>In the **Inbound rules** section, click **Add rule**.</li><li>Enter *26257* in the **Port range** field.</li><li>In the **Source** field, enter the CIDR range from Step 5.</li></ul></li>
+    <li>Click **Create security group**.</li></ul>
+{% endcapture %}
+
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+1. Select the region to create a connection in.
+
+1. Copy the **Service Name** shown in the connection modal.
 
-1. If you have a multi-region cluster, select the region to create a connection in. Skip this step if you have a single-region cluster.
-1. <a name="step-1"></a> Copy the **Service Name** shown in the connection modal.
 1. On the [Amazon VPC Console](https://console.aws.amazon.com/vpc/) in your AWS account, click **Your VPCs** in the sidebar.
-1. Locate the VPC ID of the VPC you want to create your endpoint in.
 
-    This will probably be the VPC which hosts the application or source program which is supposed to access your {{ site.data.products.dedicated }} cluster. You can also choose a different VPC as long as it is peered to the VPC your application is running in and the private endpoint is configured to be DNS-accessible across the peered VPCs.
+1. Locate the VPC ID of the VPC you want to create your endpoint in. Cockroach Labs recommends that you create the VPC in the availability zones where your cluster is deployed, and as near as possible to the availability zones where your application or service is deployed. You can also choose a different VPC as long as it is peered to the VPC your application is running in and the private endpoint is configured to be DNS-accessible across the peered VPCs.
 
 1. On the **Your VPCs** page, locate the IPv4 CIDR corresponding to the VPC you chose in Step 4.
+
 1. Click **Subnets** in the sidebar.
+
 1. Locate the subnet IDs corresponding to the VPC you chose in Step 4.
+
 1. Click **Security Groups** in the sidebar.
-1. <a name="step-8"></a> Click **Create security group** to create a security group within your VPC that allows inbound access from your application or source program on Port 26257:
-  - In the **Security group name** field, enter a name for the security group.
-  - In the **Description** field, enter a description for the security group.
-  - From the **VPC** dropdown, select the VPC you chose in Step 4.
-  - In the **Inbound rules** section, click **Add rule**. Enter *26257* in the **Port range** field. In the **Source** field, enter the CIDR range from Step 5.
-  - Click **Create security group**.
+
+1. Click **Create security group** to create a security group within your VPC that allows inbound access from your application or source program on Port 26257: {{ security_group_substeps }}
+</section>
+
+<section class="filter-content" markdown="1" data-scope="dedicated aws-console">
```

Comment:
I think the problem might be in mixing filter groups. The way the JS shows and hides sections groups on the original filter divs. So one group is dedicated/serverless, a different group is aws-console/aws-cli.